install.sh

```diff
@@ -374,6 +385,13 @@ do_install() {
 					add_debian_backport_repo "$dist_version"
 				fi
 			fi
+
+			# TODO: March 1, 2018 delete from here,
+			if [ "$lsb_dist" =  "ubuntu" ] && [ "$dist_version" = "zesty" ]; then
+				deprecation_notice "$lsb_dist $dist_version" "March 1, 2018"
+			fi
+			# TODO: March 1, 2018 delete to here,
```

Comment:
Are the TODOs addressed? 
Do you need them anymore?

Reply:
The TODO's are for the future so we remember to delete these lines when the time comes